### PR TITLE
running, tests passing

### PR DESCRIPTION
**Pull request by larshp**

### CODE REVIEW

**Comment by larshp** [comment]
🎉 ready for some review

this is a minimal approach, with multiple things that can be improved. I prefer having things working end2end before making it more complicated 🤪

**Comment by larshp** [comment]
pending #8 before merging

File: bin/run-in-docker.sh [hunk 1]
```diff
@@ -38,6 +38,7 @@ docker run \
     --network none \
     --mount type=bind,src="${solution_dir}",dst=/solution \
     --mount type=bind,src="${output_dir}",dst=/output \
+    --tmpfs /tmp:rw \
     exercism/test-runner "${slug}" "//solution" "//output"
```
Comment (ErikSchierboom) [inline bin/run-in-docker.sh:42]: Should this be (looking at the mount):

```suggestion
    exercism/test-runner "${slug}" "/solution" "/output"
```

File: bin/run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
+
+    # OPTIONAL: Normalize the results file
+    # If the results.json file contains information that changes between
+    # different test runs (e.g. timing information or paths), you should normalize
+    # the results file to allow the diff comparison below to work as expected
+    # sed -i -E \
+    #   -e 's/Elapsed time: [0-9]+\.[0-9]+ seconds//g' \
+    #   -e "s~${test_dir_path}~/solution~g" \
+    #   "${results_file_path}"
```
Comment (ErikSchierboom) [inline bin/run-tests.sh:32]: Feel free to remove this comment.

**Review by ErikSchierboom — APPROVED** [review]
This appears to be working well!

My only suggestion would be to add two test cases for failure:

1. One with _all_ tests failing
2. One with _some_ tests failing

File: bin/run-in-docker.sh (hunk 1 above)
Comment (larshp) [reply bin/run-in-docker.sh:42]: yea, its kind of a mess, I'm running bash on windows, [which does magic things](https://newbedev.com/git-bash-string-parameter-with-at-start-is-being-expanded-to-a-file-path-how-to-stop-this)

anyhow, it now runs on github actions

File: bin/run-tests.sh (hunk 2 above)
Comment (larshp) [reply bin/run-tests.sh]: removed in https://github.com/exercism/abap-test-runner/pull/7/commits/c27b70d1104ff6dd5500d331067df51214cc5a77

**Comment by larshp** [comment]
thanks, regarding additional unit tests, good point, I've opened https://github.com/exercism/abap-test-runner/issues/10 for it

**Comment by ErikSchierboom** [comment]
Feel free to merge!

**Pull request merged**